packages/server/lib/routes-ct.ts

```diff
@@ -1,61 +0,0 @@
-import Debug from 'debug'
-import { Request, Response, Router } from 'express'
-import type { InitializeRoutes } from './routes'
-import send from 'send'
-import { getPathToDist } from '@packages/resolve-dist'
-
-const debug = Debug('cypress:server:routes-ct')
-
-const serveChunk = (req: Request, res: Response, clientRoute: string) => {
-  let pathToFile = getPathToDist('runner', req.originalUrl.replace(clientRoute, ''))
-
-  return send(req, pathToFile).pipe(res)
-}
-
-export const createRoutesCT = ({
-  config,
-  nodeProxy,
-}: InitializeRoutes) => {
-  const routesCt = Router()
-
-  routesCt.get(`/${config.namespace}/static/*`, (req, res) => {
```

Comment:
I'm not seeing these routes in the consolidated `routes.ts` file - not necessary anymore?

Reply:
Finding out now, this is not really ready for review until Cy in Cy is fixed